Add-ons/UmbracoForms/Developer/Custom-Markup/index.md

```diff
@@ -65,6 +65,16 @@ Umbraco Forms uses ASP.NET Unobtrusive Validation which is why you see attribute
 
 This can be customized but it's important to keep the ID of the control to `@Model.Id` since that is used to match the value to the Form field.  For fields that are conditionally hidden, without an ID of `@Model.Id` the control won't be shown when the conditions to show the field are met.  An ID needs to be added to text controls such as headings and paragraphs.
 
+The view model for the partial view for a field is `FieldViewModel`. This defined various properties that may be useful when creating new themes or custom fields, some of which are shown in the code samples above.  Others include:
```

Comment:
The view model for the partial view of a field is `FieldViewModel`. This defines various properties that may be useful when creating new themes or custom fields, some of which are shown in the code samples above.  Others include: